packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7153,7 +7153,83 @@ class KeyedSubtree extends StatelessWidget {
   Widget build(BuildContext context) => child;
 }
 
-/// A platonic widget that calls a closure to obtain its child widget.
+/// A platonic (simple) widget that provides a specific [BuildContext] object to its child
```

Comment:
I thought we wanted to remove platonic? /cc @sfshaza2

Reply:
see previous comments

Reply:
Which one?

Reply:
https://github.com/flutter/flutter/pull/63584#discussion_r472406104

Reply:
I think we should remove "platonic" altogether, but I leave the last vote to @sfshaza2.

Reply:
Yes, please remove platonic. It's too confusing, even for folks who know English pretty well.